### DISCUSSION TITLE
How to set focus at next line.

### DISCUSSION THREAD

**Discussion by patsornk** [ueberdosis/tiptap#1800] [General Questions] [answered]
At this image below show to how to set focus. But not have solution to set focus at next line. Can I ?

![image](https://user-images.githubusercontent.com/76081944/130887553-ee5bb487-9df6-4783-9bb0-4c837cdab325.png)

**Comment by philippkuehn** [comment, marked as answer]
That’s definitely possible with calculating the needed position. [Resolved positions](https://prosemirror.net/docs/ref/#model.Resolved_Positions) (like `editor.state.selection.$from`) are super powerful for doing that.

**Reply by patsornk** [reply]
I got it. Thank for your help.
